grepos/app/src/main/java/net/boondockradio/grepos/dto/Repository.java

```diff
@@ -1,75 +1,146 @@
 
 package net.boondockradio.grepos.dto;
 
-import java.util.ArrayList;
-import java.util.List;
-import com.google.gson.annotations.Expose;
 import com.google.gson.annotations.SerializedName;
 
 public class Repository {
 
-    @SerializedName("total_count")
-    @Expose
-    private Integer totalCount;
-    @SerializedName("incomplete_results")
-    @Expose
-    private Boolean incompleteResults;
-    @SerializedName("items")
-    @Expose
-    private List<Item> items = new ArrayList<Item>();
-
-    /**
-     * 
-     * @return
-     *     The totalCount
-     */
-    public Integer getTotalCount() {
-        return totalCount;
-    }
-
-    /**
-     * 
-     * @param totalCount
-     *     The total_count
-     */
-    public void setTotalCount(Integer totalCount) {
-        this.totalCount = totalCount;
-    }
-
-    /**
-     * 
-     * @return
-     *     The incompleteResults
-     */
-    public Boolean getIncompleteResults() {
-        return incompleteResults;
-    }
-
-    /**
-     * 
-     * @param incompleteResults
-     *     The incomplete_results
-     */
-    public void setIncompleteResults(Boolean incompleteResults) {
-        this.incompleteResults = incompleteResults;
-    }
-
-    /**
-     * 
-     * @return
-     *     The items
-     */
-    public List<Item> getItems() {
-        return items;
-    }
-
-    /**
-     * 
-     * @param items
-     *     The items
-     */
-    public void setItems(List<Item> items) {
-        this.items = items;
-    }
-
+    @SerializedName("id")
+    public int id;
+    @SerializedName("name")
+    public String name;
+    @SerializedName("full_name")
+    public String fullName;
+    @SerializedName("owner")
+    public Owner owner;
+    @SerializedName("private")
+    public boolean _private;
```

Comment:
should be change the json property `private` -> `isPrivate`  if we can modify to the api response.
I think same as dto and json property name but this case is annoying.
maybe better `_private` I feel.